skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkiaLayer.jvm.kt

```diff
@@ -53,7 +53,7 @@ actual open class SkiaLayer internal constructor(
             }
         }
 
-    internal val backedLayer: HardwareLayer
+    internal var backedLayer: HardwareLayer
```

Comment:
We don't change it. So it still can be `val`?

Reply:
Done.